gluoncv/model_zoo/mobilenetv3.py

```diff
@@ -0,0 +1,314 @@
+        self.norm_kwargs = norm_kwargs
+        self.inplanes = 16
+
+        with self.name_scope():
+            self.features = nn.HybridSequential(prefix='')
+            self.features.add(nn.Conv2D(channels=make_divisible(k*self.inplanes), \
+                                        kernel_size=3, padding=1, strides=2,
+                                        use_bias=False, prefix='first-3x3-conv-conv2d_'))
+            self.features.add(nn.BatchNorm(prefix='first-3x3-conv-batchnorm_'))
+            self.features.add(HardSwish())
+            i = 0
+            for layer_cfg in cfg:
+                layer = self._make_layer(kernel_size=layer_cfg[0],
+                                         exp_ch=make_divisible(k * layer_cfg[1]),
+                                         out_channel=make_divisible(k * layer_cfg[2]),
+                                         use_se=layer_cfg[3],
+                                         act_func=layer_cfg[4],
+                                         stride=layer_cfg[5],
+                                         prefix='seq-%d'%i,
+                                        )
+                self.features.add(layer)
+                i += 1
+            self.features.add(nn.Conv2D(channels= \
+                         make_divisible(k*cls_ch_squeeze), \
+                         kernel_size=1, padding=0, strides=1,
+                                        use_bias=False, prefix='last-1x1-conv1-conv2d_'))
+            self.features.add(nn.BatchNorm(prefix='last-1x1-conv1-batchnorm_',
+                                           **({} if norm_kwargs is None else norm_kwargs)))
+            self.features.add(HardSwish())
+            self.features.add(nn.GlobalAvgPool2D())
+            self.features.add(nn.Conv2D(channels=cls_ch_expand, kernel_size=1, padding=0, strides=1,
+                                        use_bias=False, prefix='last-1x1-conv2-conv2d_'))
+            self.features.add(HardSwish())
+
+            if final_drop > 0:
+                self.features.add(nn.Dropout(final_drop))
+            self.output = nn.HybridSequential(prefix='output_')
+            with self.output.name_scope():
+                self.output.add(
+                    nn.Conv2D(in_channels=cls_ch_expand, channels=classes,
+                              kernel_size=1, prefix='fc_'),
+                    nn.Flatten())
+
+
+    def _make_layer(self, kernel_size, exp_ch, out_channel, use_se, act_func, stride=1, prefix=''):
+
+        mid_planes = exp_ch
+        out_planes = out_channel
+        layer = _ResUnit(self.inplanes, mid_planes, \
+                         out_planes, kernel_size, \
+                         act_func, strides=stride, use_se=use_se, prefix=prefix)
+        self.inplanes = out_planes
+        return layer
+
+    def hybrid_forward(self, F, x):
+        x = self.features(x)
+        x = self.output(x)
+        return x
+
+
+def get_mobilenetv3(model_name, multiplier=1., **kwargs):
```

Comment:
we need the same set of arguments as in https://github.com/dmlc/gluon-cv/blob/master/gluoncv/model_zoo/mobilenet.py#L292 : `pretrained`, `ctx` and `root`. Please update the definition according to the mobilenetv2 example.

These arguments help to prepare this function ready for loading pretrained params.